docs/src/extensions.md

```diff
@@ -51,6 +51,17 @@ The [`@constraint`](@ref) macro always calls the same three functions:
 Adding methods to these functions is the recommended way to extend the
 [`@constraint`](@ref) macro.
 
+### Adding `parse_constraint` methods
+
+```@meta
+# TODO(Benoît): Details how `parse_constraint` works and shows how
```

Comment:
Grammar nit: "Detail how `parse_constraint` works and show how `sense_to_set` fits into the picture."